### DISCUSSION TITLE
Could we add "non-NYU community member" to the "select role" drop down menu on the booking page?

### DISCUSSION THREAD

**Discussion by mab261** [ITPNYU/booking-app#242] [General]

**Comment by nopivnick** [comment]
hi, Michael. thanks for the suggestion. my understanding is (at least for the time being) non-NYU community members won't be submitting requests themselves but rather you will be submitting on their behalf via the booking tool. for that matter, the Media Commons website is only accessible by the NYU community so non-NYU community members won't be able to get to the booking tool anyway. @ss12430 should confirm.
